keyboards/ergodox_ez/build.mk

```diff
@@ -0,0 +1,3 @@
+ifeq ($(strip $(RGB_MATRIX_ENABLE)), no)
```

Comment:
This change isn't actually necessary anymore.  

i2c is added via `QUANTUM_LIB_SRC` which has some special handling that prevents conflicts with it being added multiple times (also, prevents LTO from being applied to it, as it can cause weird timing related issues)

Reply:
oh, neat. that would indeed resolve this part. i'm not sure it also resolves the corresponding issue with the other optional modules, added in the later patch, though?

Reply:
Yeah.   There are still a number of use cases that would make this very useful still.  Such as per-keyboard features/options, that can be adjusted by the keymaps.  As well as by userspace, in general.